src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -344,6 +345,9 @@ private PaintingSession CreateLocalSession(in PaintingSession parentSession)
 		return session;
 	}
 
+	/// <remarks>This does NOT take the clipping into account.</remarks>
+	internal virtual bool HitTest(Point point) => false;
```

Comment:
Somewhat weird to be in Visual but not all visuals have proper override. Maybe not reachable for those visuals? In this case maybe we can throw here or at least add a failing assert

Reply:
Yeah, you're right. Definitely not reachable. I thought I would move the logic to be inside TextVisual instead of the TextBlock itself, but because the TextVisual is not `TextBlock.Visual`, but a child of it, it doesn't work out.

Reply:
In that case let's keep it specifically to ShapeVisual